app/models/manageiq/providers/redhat/infra_manager/event_fetcher.rb

```diff
@@ -313,6 +313,7 @@ def set_event_name!(event)
     602  => "USER_HOST_MAINTENANCE_MIGRATION_FAILED",
     603  => "HOST_SET_NONOPERATIONAL_IFACE_DOWN",
     605  => "HOST_USED_AS_PROXY",
+    620  => "USER_HOST_MAINTENANCE",
```

Comment:
Good idea, I haven't seen those events but will add those here. I think that as some point we need to align this file with [1]. I will add a card to our backlog.

[1] https://github.com/oVirt/ovirt-engine/blob/master/backend/manager/modules/common/src/main/java/org/ovirt/engine/core/common/AuditLogType.java